doc/source/python/python_wrapping_docker.md

```diff
@@ -200,9 +200,9 @@ class MyModel(object):
     	return [{"type": "COUNTER", "key": "mycounter", "value": 1}]
 ```
 
-For more details on custom metrics and the format of the metric dict see [here](../custom_metrics.md).
+For more details on custom metrics and the format of the metric dict see [here](https://docs.seldon.io/projects/seldon-core/en/latest/analytics/analytics.html#custom-metrics).
```

Comment:
Can we try to ensure all internal docs links remain relative as that ensures they will link to the same version when published wheras the above external link would move people to the latest version.